src/rust/ensogl/src/display/render/passes/symbols.rs

```diff
@@ -31,16 +31,21 @@ impl SymbolsRenderPass {
 
 impl RenderPass for SymbolsRenderPass {
     fn outputs(&self) -> Vec<RenderPassOutput> {
-        vec![ RenderPassOutput::new("color",texture::Rgba,texture::item_type::u8)
-            , RenderPassOutput::new("id",texture::Rgba32ui,texture::item_type::u32)
+        let color_parameters = texture::Parameters::default();
+        let id_parameters    = texture::Parameters {
+            min_filter : texture::MinFilterValue::NEAREST,
+            mag_filter : texture::MagFilterValue::NEAREST,
+            ..Default::default()
```

Comment:
Please clean the codebase and then run Clippy on it. It should catch that you don't need qualification here :)

Reply:
`node run lint` does not show anything here. Neither does the CI. But it's removed now.